qc-runner/src/cellSizeDistribution.r

```diff
@@ -50,7 +50,7 @@ generate_default_values_cellSizeDistribution <- function(seurat_obj, config) {
   return(sample_subset$nCount_RNA)
 }
 
-task <- function(seurat_obj, config, task_name, sample_id) {
+task <- function(seurat_obj, config, task_name, sample_id, num_cells_to_downsample = 5000) {
```

Comment:
5000 is the default, but if other values are desired, how are they controlled? Via DynamoDB?

Reply:
Currently, these values are going to be tested by @aerlaut, but for now they are HARDCODED. For a long-term we need to think how we control them. Maybe we can add an entry in the config

Reply:
Ok, just wanted to know the state, fine for me for the time being.

Reply:
I've checked and the uploaded file plot data file contains 5000 samples. The UI is still slow though,